README.md

```diff
@@ -341,6 +342,25 @@ The Puppet agent on a Windows node can run DSC with a normal default install. If
 * The puppet types are built from the source code of each DSC Resources MOF schema files.
 * If you want the build Puppet types for your own custom DSC Resources, read the [README_BUILD](README_BUILD.md)
 
+## Places to Learn More About DSC
+
+You can learn more about PowerShell DSC from the following online resources:
+
+- [TechNet Microsoft PowerShell Desired State Configuration Overview](https://technet.microsoft.com/en-us/library/dn249912.aspx)
```

Comment:
Sorry ... one last other minor nit @jpogran - could you adjust the link texts here ... sometimes you have the title of the thing hyperlinked, and other times you have the word `book`, `blog`, `webinar` or `talk` hyperlinked.  

I'm cool with whatever you think looks best, but I think it would good to be consistent.